test/test_instructions.py

```diff
@@ -129,31 +133,61 @@ def both():
         left()
         screen("exit")
         left()
+        screen("settings")
+        left()
         screen("high_watermark")
-    else:
+        right()
+        # Check settings menu
+        screen("settings")
+        both()
+        screen("hwm_status_enabled")
+        both()
+        screen("hwm_status_disabled")
+        right()
+        screen("back")
+        both()
+        screen("home_screen")
+        left()
+        screen("exit")
+        right()
         screen("home_screen")
+        left()
+    else:
+        backend.wait_for_home_screen()
         tezos_navigator.home.settings()
         backend.wait_for_screen_change()
         screen("app_context")
-        tezos_navigator.settings.next()
+        tezos_navigator.sub_settings.next()
+        backend.wait_for_screen_change()
+        screen("description")
+        tezos_navigator.sub_settings.next()
+        backend.wait_for_screen_change()
+        screen("hwm_status")
+        tezos_navigator.sub_settings.previous()
         backend.wait_for_screen_change()
         screen("description")
-        tezos_navigator.settings.previous()
+        tezos_navigator.sub_settings.previous()
         backend.wait_for_screen_change()
         screen("app_context")
         tezos_navigator.settings.multi_page_exit()
-        backend.wait_for_screen_change()
-        screen("home_screen")
+        backend.wait_for_home_screen()
         tezos_navigator.home.settings()
         backend.wait_for_screen_change()
         screen("app_context")
-        tezos_navigator.settings.next()
+        tezos_navigator.sub_settings.next()
+        backend.wait_for_screen_change()
+        screen("description")
+        tezos_navigator.sub_settings.next()
+        backend.wait_for_screen_change()
+        screen("hwm_status")
+        # TODO: switch hwm status off and check the screen.
```

Comment:
I think we should check the `hwm` toggle here (we can probably use `Layouts.ChoiceList`)